lib/utils/telemetry/generatePayload.js

```diff
@@ -128,7 +129,7 @@ const getServiceConfig = ({ configuration, options, variableSources }) => {
 
 // This method is explicitly kept as synchronous. The reason for it being the fact that it needs to
 // be executed in such manner due to its use in `process.on('SIGINT')` handler.
-module.exports = ({
+module.exports = async ({
```

Comment:
This method needs to stay _sync_, see comment you've added above previously  :)

Reply:
Changed the approach :+1: